tests/console-backend-service/internal/domain/serverless/serverless.go

```diff
@@ -0,0 +1,8 @@
+package serverless
+
+const (
+	ModuleName         = "serverless"
```

Comment:
You may not export those vars, because you use them only in `serverless` domain :)

Reply:
done